source/WB-CAN-Firmware.c

```diff
@@ -304,21 +448,17 @@ float adcRead(adc16_config_t adc16ConfigStruct, adc16_channel_config_t adc16Chan
 		adcValue = 0; //65536U - adcValue;
 	}
 
-
-	// Duty Cycle from 0-100 Used for Potentiometer Setup:
-	//adcValue = adcValue / 4096.0 * 100.0; // Reads in Percentage
-	adcValue = adcValue / 4096.0 * 3.3  ; // Reads in Voltage
-
-	// When Reading from Pressure Sensor -  Double check this conversion
-	adcValueATM = adcValue * 3.30;
+	// Maps reading to Voltage
+	adcValue = adcValue / 4096.0 * 3.3  ;
```

Comment:
Can we define these magic numbers as compiler defs? Otherwise it's pretty mystical to anyone else.

Reply:
Will do in the header file branch as I've already moved it.